inference-engine/tests/functional/plugin/gna/pass_tests/convert_matmul_to_fullyconnected.cpp

```diff
@@ -99,7 +99,8 @@ const std::vector<std::vector<std::vector<size_t>>> input_shapes = {
         {{1, 8}, {8, 1}},
```

Comment:
This file name is quite confusing, actually not all test parameters cause MatMul conversion to FullyConnected.
I'd rename this file yo something like convert_matmul_with_const_first_input.cpp and create 2 separate test suites for cases when MatMul will be converted to FullyConnected (inputs swap transformation will be applied) and Gemm (inputs swap transformation won't be applied).

Reply:
done